paddle/fluid/operators/activation_op.h

```diff
@@ -37,6 +37,8 @@ limitations under the License. */
 namespace paddle {
 namespace operators {
 
+using framework::To32BitIndex;
```

Comment:
why do you change the file mode 100644 → 100755

Reply:
have changed it back.